lib/rspec/core/example_group.rb

```diff
@@ -33,6 +33,8 @@ class ExampleGroup
       include Pending
       extend SharedExampleGroup
 
+      # Define a singleton method for the eigenclass (remove the method if
+      # it's already been defined).
```

Comment:
Are you using this method some how?  It's private so it's not meant to be used outside of rspec-core itself.  We're always happy to improve documentation, though (even though this won't be published with our docs since it's private).  I'd ask that you use "singleton class" instead of "eigenclass", though, since the latter isn't commonly used in the Ruby community anymore (particularly since `singleton_class` got added as an API to Ruby 1.9).